contracts/atomic-swap/src/contract.rs

```diff
@@ -0,0 +1,619 @@
+use cosmwasm_std::{
+    log, to_binary, Api, BankMsg, Binary, Coin, CosmosMsg, Env, Extern, HandleResponse, HumanAddr,
+    InitResponse, Querier, StdError, StdResult, Storage,
+};
+use sha2::{Digest, Sha256};
+
+use cw2::{set_contract_version, ContractVersion};
+
+use crate::msg::{
+    is_valid_name, CreateMsg, DetailsResponse, HandleMsg, InitMsg, ListResponse, QueryMsg,
+};
+use crate::state::{all_swap_ids, atomic_swaps, atomic_swaps_read, AtomicSwap};
+
+// Version info, for migration info
+const CONTRACT_NAME: &str = "crates.io:atomic-swap";
+const CONTRACT_VERSION: &str = "v0.1.0";
```

Comment:
I realized this is prone to not be updated (I left it 0.1.0 on many contracts for the 0.1.1 release).

Please use: `env!("CARGO_PKG_VERSION");` (see: https://github.com/CosmWasm/cosmwasm-plus/blob/master/contracts/cw1-whitelist/src/contract.rs#L16)